packtools/sps/validation/journal_meta.py

```diff
@@ -166,11 +166,47 @@ def validate_publisher_names(self, publisher_name_list):
         if not publisher_name_list:
             raise ValidationPublisherException("Function requires a list of publisher names")
 
+        expected = publisher_name_list
+        obtained = self.publisher.publishers_names
 
+        for publisher in zip(expected, obtained):
+            is_valid = publisher[0] == publisher[1]
             yield {
                     'title': 'Publisher name element validation',
                     'xpath': './/publisher//publisher-name',
                     'validation_type': 'value',
+                    'response': 'OK' if is_valid else 'ERROR',
+                    'expected_value': publisher[0],
+                    'got_value': publisher[1],
+                    'message': 'Got {} expected {}'.format(publisher[1], publisher[0]),
+                    'advice': None if is_valid else 'The publisher name in the XML does not match what was expected, '
+                                                    'add {} in the XML'.format(publisher[0])
+            }
+
+        if len(obtained) != len(expected):
+            if len(expected) > len(obtained):
+                diff = expected[len(obtained):]
+                item_description = 'not found'
+                action = ('Complete', 'in')
+            else:
+                diff = obtained[len(expected):]
+                item_description = 'surplus'
```

Comment:
use `are not expected` no lugar de `surplus`

Reply:
OK, @robertatakenaka